css.js

```diff
@@ -39,10 +39,24 @@ if (typeof window !== 'undefined') {
             resolve('');
         }, 7);
       };
-      var link = document.createElement('link');
-      link.type = 'text/css';
-      link.rel = 'stylesheet';
-      link.href = url;
+      
+      var present = false;
+      var link = undefined;
+      
+      for (var i = 0; i < links.length; i++) {
+        if (links[i].href.split('?')[0] === url) {
```

Comment:
If the CSS path already had a `?` in the name this would break. If we do re-injection, then we can set an attribute on the link tag itself, say something like `systemjs-css`. That attribute can then indicate which link tags to do reinjection on (and avoid the collision detection system that is already in place).